docs/cryptodoc/src/90_bibliographie.rst

```diff
@@ -26,10 +26,23 @@
    "Message verification and transmission error detection by block chaining",
    US Patent 4074066, 1976
 
+.. [CBits] Daniel J. Bernstein:
+   "Verified fast formulas for control bits for permutation networks",
+   2020
+
 .. [CCM] U.S. DEPARTMENT OF COMMERCE/National Institute of Standards and Technology:
    "Recommendation for Block Cipher Modes of Operation: The CCM Mode for Authentication and Confidentiality",
    July 2007
 
+.. [CMCE-ISO] ISO Draft (2023):
+   Information security — Encryption algorithms — Part 1978:
```

Comment:
Make consistent with FrodoKEM ISO? This seems like the ISO draft has been submitted, but afaik it hasn't.

Also, add authors (CMCE Team)